content/pages/meetup/2022/10-nycu.rst

```diff
@@ -0,0 +1,104 @@
+
+Sciwork hosted `a panel discussion in May <https://sciwork.dev/sprint/2022/05-nycu-career>`__. 
+The panel discussion in this meetup will feature the same theme, but bring new perspectives from 
+different Panelists_.
+
+You can join both the panel and the sprint or either of them. Put your name and contact in the 
+hackmd document (`Sign Up`_). Online discussions are available at https://discord.gg/6MAkFrD .
+
+
+Panelists
+---------
+
+Dr. Shuen-Huei “Drake” Guan (https://www.linkedin.com/in/drakeguan/) is an expert in streaming and computer graphics. 
+He is currently the Chief Technology Officer of KKStream, a leading company in cloud streaming, AI, and data analytics. 
+Drake is enthusiastic about technology, and spends a lot of time in the communities of Taipei Video Tech and PyCon Taiwan to share what he thinks about and promote technical development.
+
+Chun-Yu “Joe” Tseng (https://www.linkedin.com/in/excusemejoe/) is a VP of Berry AI, a company offering advanced computer vision and AI technologies to help fast food chains grow and perform. 
+Joe is also a Python lover and expert of devops and AI products. He leads teams to provide solutions for real-world problems, which are oftentimes edible. 
+Joe has a lot of experience in both technical development and community, and has served as the chairperson of PyCon Taiwan in 2020.
+
+
+Moderator
+---------
+
+Dr. Yung-Yu Chen (https://www.linkedin.com/in/yungyuc/) is a R&D engineer
+developing software for numerical and technical applications.  He is currently
+working on mask synthesis at Synopsys and teaching a course at NYCU.
+
+
+Sign Up
+-------
+
+Add your name to the sprint note: https://hackmd.io/@sciwork/meetup-2210 .
+
+
+Agenda
+------
+
+Date: 14th October, Friday, 2022
+
+- 13:30 – 13:50 Attendee Arrival
+- 13:50 – 14:00 Opening and Introducing panelists
+- 14:00 – 15:30 Career Panel
+- 15:30 – 16:00 Refreshment
+- 16:00 – 18:00 Project Sprint
+- 18:00 – 19:00 Project Presentation
+- 19:00 End of Event / Off for Dinner
+
+
+Sprint Projects and Requirements
+--------------------------------
+
+If you want to join and bring your projects, please `Contact us`_.  Please
+follow the links for what are required for each of the projects:
+
+* modmesh: https://github.com/solvcon/modmesh
+
+.. Sponsors
+.. --------
+
+Venue
+-----
+
+`國立陽明交通大學 工程三館 329 室 <https://goo.gl/maps/pKcyhPeJTJS11hNz8>`__
+
+| Address: EC329 No. 1001, University Road, Hsinchu 300, Taiwan
+|   地址: 新竹市東區大學路 1001 號 工程三館 329 室
+|
+
+.. raw:: html
+
+  <div style="overflow:hidden; padding-bottom:56.25%; position:relative; height:0;">
+    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3622.243194966577!2d120.99445851483667!3d24.78712468408855!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3468360f96adabd7%3A0xedfd1ba0fa6c6bf7!2z5Lqk6YCa5aSn5a245bel56iL5LiJ6aSo!5e0!3m2!1szh-TW!2stw!4v1663061149092!5m2!1szh-TW!2stw" 
+        width="600" height="450" style="border:0;" allowfullscreen="" 
```

Comment:
Could you make the embedded map to be as wide as the page?

Reply:
Sure, let me change it.